src/main/java/network/brightspots/rcv/BaseCvrReader.java

```diff
@@ -65,6 +77,30 @@ Map<String, Integer> gatherUnknownCandidates(List<CastVoteRecord> castVoteRecord
       }
     }
 
+    if (includeCandidatesWithZeroVotes) {
+      // Second pass: read all candidates
+      // Not all CVR Readers have this implemented, so it's just a catch to
+      // find any candidate with zero votes.
+      List<String> allCandidates = new ArrayList<>();
+      try {
+        allCandidates = readCandidateListFromCvr(castVoteRecords);
```

Comment:
do we also need to do the `config.getNameForCandidate(candidateName) != null` check down here?